sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -904,6 +904,6 @@ def _handle_features_sbol_to_gb(self, seq_rec: SeqRecord,
 
         # Sort features based on feature location start/end, lexicographically, and then by
         # strand / number of qualifiers / type of feature string comparison
-        seq_rec_features.sort(key=lambda feat: (feat_order[feat], feat.strand, len(feat.qualifiers), feat.type))
```

Comment:
Is the feat_order variable now used anywhere? If not, then it should probably be removed too.

Reply:
Good catch. I have removed the variable in a new commit.